### PR TITLE
Add i18next to dev page

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: locales/en-US/demographic_enums.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "Total": "Total",
+  "Male": "Masculino"
```
Comment (shifucun) [inline locales/en-US/demographic_enums.json:3]: this is not en-US?

File: static/js/chart/draw.ts [hunk 2]
```diff
@@ -26,6 +26,60 @@ import {
   formatYAxisTicks,
 } from "./base";
 
+import i18next from "i18next";
+import Backend from "i18next-http-backend";
+import Cache from "i18next-localstorage-cache";
+import postProcessor from "i18next-sprintf-postprocessor";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18next
```
Comment (shifucun) [inline static/js/chart/draw.ts:35]: should this init be done in the javascript entry point? like place.ts?

File: static/js/chart/draw.ts [hunk 3]
```diff
@@ -57,6 +111,15 @@ const AXIS_GRID_FILL = "#999";
 // Max Y value used for y domains for charts that have only 0 values.
 const MAX_Y_FOR_ZERO_CHARTS = 10;
 
+function translate(text: string): string {
```
Comment (shifucun) [inline static/js/chart/draw.ts:114]: nit: name the function "translated"

File: static/js/chart/draw.ts [hunk 4]
```diff
@@ -57,6 +111,15 @@ const AXIS_GRID_FILL = "#999";
...
+  if (i18next.t(text)) {
+    text = i18next.t(text);
+  }
+  return text;
+}
+function translate_key(key: { label: string; link?: string }) {
```
Comment (shifucun) [inline static/js/chart/draw.ts:120]: nit: can make this type an interface and have a member function "translate()"

File: static/js/chart/draw.ts (hunk 3 above)
Comment (tjann) [reply static/js/chart/draw.ts]: I see, to reduce conflict with d3. Done.

File: static/js/chart/draw.ts (hunk 4 above)
Comment (tjann) [reply static/js/chart/draw.ts]: I realize this is actually not needed. Now it's just translated

**Review by shifucun — APPROVED** [review]
very impressive pioneer work!

**Comment by tjann** [comment]
Was able to block translation for local in d648d70 by returning '' instead of the json file contents when FLASK_ENV != prod. I changed it to == prod before commiting.

**Pull request merged**